components/announcement_bar/configuration_bar/configuration_bar.jsx

```diff
@@ -176,9 +211,13 @@ class ConfigurationAnnouncementBar extends React.PureComponent {
                     />
                 );
             }
-            if (this.props.license.IsLicensed === 'false' && this.props.warnMetricsStatus) {
+            if (this.props.license &&
+                this.props.license.IsLicensed === 'false' &&
```

Comment:
Can we use optional chaining here? Not sure if it works or not
```suggestion
                this.props.license?.IsLicensed === 'false' &&
```
Maybe?

Reply:
The current pattern in the code is the one being used here as well.